functions/Test-DbaBackupInformation.ps1

```diff
@@ -1,170 +1,171 @@
-                    }
-                    elseif ($path -in $OtherFileCheck){
-                        Write-Message -Message "File $Path already exists on $SqlInstance and owned by another database, cannot restore" -Level Warning
-                        $VerificationErrors++
-                    }
-                }
-                else {
-                    if (!(Test-DbaSqlPath -SqlInstance $RestoreInstance -Path (Split-path $path)) ){
-                        $ConfirmMessage = "`n Creating Folder $(Split-Path $path) on $SqlInstance `n"                  
-                        If ($Pscmdlet.ShouldProcess("$Path on $SqlInstance `n `n", $ConfirmMessage)) {
-                            if (New-DbaSqlDirectory -SqlInstance $RestoreInstance -Path (Split-path $path)){
-                                Write-Message -Message "Created Folder $(Split-path $path) on $SqlInstance" -Level Verbose
-                            }
-                            else {
-                                Write-Message -Message "Failed to create $(Split-path $path) on $SqlInstance" -Level Warning
-                                $VerificationErrors++
-                            }
-                        }
-                    }
-                }
-            }
-
-            #Test all backups readable
-            $allpaths = $DbHistory | Select-Object -ExpandProperty FullName
-            $allpaths_validity = Test-DbaSqlPath -SqlInstance $RestoreInstance -Path $allpaths
-            Foreach($path in $allpaths_validity) {
-                if ($path.fileexists -eq $false) {
-                    Write-Message -Message "Backup File $path cannot be read" -Level Warning
-                    $VerificationErrors++
-                }
-            }
-            #Test for LSN chain
-            if ($true -ne $Continue){   
-                if (!($DbHistory | Test-DbaLsnChain)) {
-                    Write-Message -Message "LSN Check failed" -Level Verbose
-                    $VerificationErrors++
-                }
-            }
-            if ($VerificationErrors -eq 0){
-                Write-Message -Message "Marking $Database as verified" -Level Verbose
-                $InternalHistory | Where-Object {$_.Database -eq $Database} | ForEach-Object {$_.IsVerified = $True}
-            }
-            else{
-                Write-Message -Message "Verification errors  = $VerificationErrors - Has not Passed" -Level Verbose
-            }
-        }
-        $InternalHistory
-    }
+	[CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = "Low")]
+	param (
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[object[]]$BackupHistory,
+		[Alias("ServerInstance", "SqlServer")]
+		[DbaInstanceParameter]$SqlInstance,
+		[PSCredential]$SqlCredential,
+		[switch]$Withreplace,
+		[switch]$Continue,
+		[switch]$EnableException
+	)
+
+	begin {
+		try {
+			$RestoreInstance = Connect-SqlInstance -SqlInstance $SqlInstance -SqlCredential $SqlCredential
```

Comment:
We can fix this in another PR but this should be `$server`, that is the pattern we use in every command and not sure why it was allowed in this one.